shared/message/platform/webots/messages.proto

```diff
@@ -83,20 +85,40 @@ message Message {
     string      text         = 2;
 }
 
+message VisionGroundTruth {
+    /// Indicates if this message exists
+    bool exists = 1;
+    /// Vector from the world to the ball in world space.
+    Vector3 rBWw = 5;
+    /// Vector from the world to the field in world space.
+    Vector3 rFWw = 6;
```

Comment:
```suggestion
    vec3 rBWw = 5;
    /// Vector from the world to the field in world space.
    vec3 rFWw = 6;
```

And do the same on the NUWebots side. Will make things easier since it can be used directly as an Eigen type.